CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 ### 0.9.2 (Next)
-
+* [#163](https://github.com/slack-ruby/slack-ruby-client/pull/164): Use `OpenSSL::X509::DEFAULT_CERT_DIR` and `OpenSSL::X509::DEFAULT_CERT_FILE` for default ca_cert and ca_file [@leifcr](https://github.com/leifcr)
```

Comment:
Make this in the same format as everything else, needs a `-` before your name and a `.` at the end.